src/test/unit/java/godaddy/registry/jtoolkit2/se/fee/DomainCheckIeftIdnCommandExtensionTest.java

```diff
@@ -0,0 +1,115 @@
+package godaddy.registry.jtoolkit2.se.fee;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.fail;
+
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.xml.sax.SAXException;
+
+import godaddy.registry.jtoolkit2.ErrorPkg;
+import godaddy.registry.jtoolkit2.Timer;
+import godaddy.registry.jtoolkit2.se.CLTRID;
+import godaddy.registry.jtoolkit2.se.Command;
+import godaddy.registry.jtoolkit2.se.DomainCheckCommand;
+import godaddy.registry.jtoolkit2.se.idn.ietf.DomainCheckIeftIdnEPPCommandExtension;
+
+
+public class DomainCheckIeftIdnCommandExtensionTest {
```

Comment:
Related with my previous comment - could you please 
1. Rename the existing test class `DomainCreateIetfIdnCommandExtensionTest` to `IetfIdnCommandExtensionTest`, so that the test will match the renamed main code?
2. Remove this new test class, and add the new test cases to the existing-and-renamed `IetfIdnCommandExtensionTest`.
    - Please keep those existing test cases (related with domain create) untouched.
    - You may want to rename the existing private method `expectedXmlWithExtension` to `expectedDomainCreateXmlWithExtension`
    - And for your new test cases, you may want to introduce a similar private method like  `expectedDomainCheckXmlWithExtension`, so that your new test cases would be shorter and more readable.

Reply:
Done